### ISSUE TITLE
Landing Website: add alt text to images

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#48]
for example, missing alt text on many images on the Team page
https://onlybansgame.com/wp/team/

Lena wrote alt text for many of these images already for the game
